src/phrases/phrases-nb.ts

```diff
@@ -1096,6 +1096,8 @@ const phrases = {
         row: 'rad',
         column: 'kolonne',
       },
+      normalizedOnLoad:
+        'Endringer etter oppdatering av editor, republiser artikkelen for å få med endringer',
```

Comment:
Nokke sånt er litt tydeligere
```suggestion
        'Artikkelen har endringer på grunn av oppdatering av editoren. Eventuelle endringer som lagres må republiseres.',
```